src/player.cpp

```diff
@@ -12942,6 +12976,23 @@ int player::get_hp_max( hp_part bp ) const
     return hp_total;
 }
 
+int player::get_stamina_max()
+{
+    return 1000;
```

Comment:
Hmm.  I'd like to have stronger folks have more endurance, since we don't differentiate that out of ST.  (And it would make DX less the UberStat for melee purposes.)